apps/full-site-editing/full-site-editing-plugin/dotcom-fse/editor/style.scss

```diff
@@ -35,6 +35,45 @@
 		}
 	}
 
+	// Taken from styles/starter-page-templates-editor.scss
+	$breakpoint-mobile: 660px;
+	$breakpoint-tablet: 783px;
+
+	.edit-post-header__toolbar {
+		// The right side of the header has 16px,
+		// the left side should too, otherwise it looks out of balance.
+		// Applies to all screen sizes.
+		padding-left: 16px;
+	}
+
+	@media screen and ( max-width: $breakpoint-tablet ) {
+		// Removes whitespace above header in mobile & tablet view.
+		.block-editor-editor-skeleton {
+			top: 0;
+		}
+
+		.edit-post-header__toolbar {
+			// Back button is not displayed in mobile & tablet view.
+			.edit-post-fullscreen-mode-close__toolbar__override {
```

Comment:
If we go to `dotcom-fse/plugins/close-button-override`, there is a scss file in there where changes to this buttons visibility may make more sense.

Reply:
Thanks! Fixes in this commit. https://github.com/Automattic/wp-calypso/pull/40361/commits/e6d5287ea0e0dac6e594b0d99319c54a5825e541